lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -625,8 +626,16 @@ def get(self, request, *args, **kwargs):
             try:
                 course = get_course_by_id(course_id)
             except Exception:
-                log.info('Cannot find course %s', course_id)
-                raise Http404
+                course_key = CourseLocator(
```

Comment:
This fixes issue for old imported course whose course id in mongo is in format "course-v1:org+course+run" but has course id saved in CourseImportLog as splashed, during the import .